.circleci/config.yml

```diff
@@ -405,7 +421,7 @@ workflows:
           # <<: *on-version-tags
       - deploy-snapshot:
           <<: *after-platform-builds
-          <<: *on-master
+          <<: *on-integ-branch
```

Comment:
why do we want `integ` branch snapshots? will it overwrite existing versions?

Reply:
like we get a master snapshot, we may want integration branch snapshots, with corresponding names, like `redisai/snapshots/redisai-cpu-onnxruntime.linux-bionic-x64.1.2.tgz`.
we can limit ourselves to `on-master` at the moment.